config-generator/src/generate.js

```diff
@@ -260,7 +266,8 @@ function prometheusConfig(params) {
 
             obj['remote_write'] = [{
               url: 'http://localhost:9201/write',
-              remote_timeout: '30s'
+              remote_timeout: '30s',
+              queue_config: queue_config
```

Comment:
We need to place `queue_config` inside `remote_write` and not in the global config [like I tried before](https://github.com/livepeer/livepeer-monitoring/pull/165/files)